src/api/app/models/event_subscription/for_event_form.rb

```diff
@@ -0,0 +1,16 @@
+class EventSubscription
+  class ForEventForm
+    attr_reader :event_class, :subscriber, :roles
+
+    def initialize(event, subscriber)
+      @subscriber = subscriber
+      @event_class = event
+      @roles = []
+    end
+
+    def call
+      @roles = @event_class.receiver_roles.map { |role| EventSubscription::ForRoleForm.new(role, event_class, subscriber).call }
```

Comment:
Since there's `attr_reader :event_class`, we should use `event_class`.

Reply:
Done!